packages/web/server/queries/complex/pools/providers/indexer.ts

```diff
@@ -33,11 +33,14 @@ import { DEFAULT_VS_CURRENCY } from "../../assets/config";
 import { Pool } from "..";
 import { TransmuterPoolCodeIds } from "../env";
 
-const poolsCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
+const poolsCache = new LRUCache<string, CacheEntry>({ max: 1 });
```

Comment:
Does this address the edge memory issue?

Reply:
Yes, it does. Furthermore, we only need one cache entry for the pools function because this cache won't be shared with other cached functions